source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+
+- the corresponding GitHub help wanted issue is closed, thanking the contributor
+- the corresponding JIRA ticket is resolved
+
+----
+
+Community Forums
+---------------------------------------------------------
+
+*CM = community manager*
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Support (answer everything correctly)
+- Prioritize (focus help on those who help others)
+- Empower (give them time to answer)
+- Elevate (thank, recognize and approve their work)
+- Grow (invite people to help as experts, promote people)
+
+Response timeline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Things only Mattermost knows: Respond right away (such as bugs in recent release)
+- Enterprise Edition issues: Respond right away, as these are customers or potential customers. CM posts to Community channel.
+- Places where community can help:
+  - Wait 24 hours then CM nudges community or routes (feature idea, troubleshooting, etc.)
+  - If 48 hours go by without community response, CM nudges Contributors channel (which includes core team) by asking for someone to help answer
+  - If 72 hours go by and no one in Contributors room has stepped up, then CM (at)-mentions the PM who's managing community issues that week, who can take ownership of responding or routing to someone either in community or core team.
+- On holidays, let the community step up
+
+Response writing tips
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Don't answer if unsure
+  - Ask someone who knows for sure instead of replying with an assumption or incomplete understanding
+  - Don't be afraid to re-route if you don't have the answer and are having trouble figuring it out
+- Don't make promises
+  - Don’t say “we’ll work on it” or something similar that sets expectations that aren’t met later (e.g. after presenting to core team it turns out you can’t do it)
+  - Be careful saying “that’s a good idea”, don’t just say it to be polite. Instead say something akin to “thanks for the idea”
+- Choose positivity over negativity
+  - Avoid excuses like “we’re busy”, or “our team is small” and turn a missing feature into an invitation to share a feature idea to be upvoted
+- Do your best to link documentation as answers
+  - Allows answers to be easily updated dynamically as documentation is updated
+  - Any questions that should be answered in docs that aren’t should turn into tickets to create that documentation (and post ticket in response)
+- Be thankful
+  - Communities really respond well to being praised and thanked for their work
+  
+Resolving issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- After responding to an issue, if the asker comes back and lets you know that the issue is solved, please add [Solved] to the beginning of the title of the question
+  - Don't be afraid to close GitHub issues if you think you've solved them
```

Comment:
Is this covering GitHub Issues, forums, or both? The top of the doc says this is about forums, and I'm not sure that we put solved in the GitHub issue title